sdks/java/extensions/protobuf/src/main/java/org/apache/beam/sdk/extensions/protobuf/ProtoSchemaTranslator.java

```diff
@@ -107,8 +108,8 @@
  *   <li>google.protobuf.BytesValue maps to a nullable FieldType.BYTES
  * </ul>
  */
-@Experimental(Experimental.Kind.SCHEMAS)
-public class ProtoSchemaTranslator {
```

Comment:
It think that this need to be kept public as well. Not sure though... it can but used externally to create a schema from a descriptor.

Reply:
This one looks like an Internal util class no?
Users who need to create an explicit Schema at the moment can get it via `ProtoMessageSchema#schemaFor`
WDYT of letting it Internal and exposing it only once required?